### DISCUSSION TITLE
[request] Capping Plugin

### DISCUSSION THREAD

**Discussion by sonicaghi** [Kong/kong#7354] [Ideas and feature requests]
Limit # of requests / calls / etc to a specific number in a consumer lifetime.

Non-Interval and Interval based.

Examples:
- 23 Requests per Month.
- 10 Requests ever.

**Comment by subnetmarco** [comment]
What is this?

**Comment by sonicaghi** [comment]
hard stop on rate limiting and not per timeframe. Like 10 calls max lifetime.

**Comment by nijikokun** [comment]
I've updated @sinzone title and message to better convey the request.

**Comment by thibaultcha** [comment]
The idea of this was to integrate it to the ratelimiting plugin tho. So should it be a new plugin or a new limit (hence, `overall`) in the ratelimiting plugin?

**Comment by subnetmarco** [comment]
+1 that it should be a period, like `alltime`, along with `second`, `minute`, etc.

**Comment by lucamaraschi** [comment]
:+1:
